tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -158,6 +161,12 @@ def parse_args():
                              'NNAPI execution provider takes, in order to preserve those nodes in the ORT format '
                              'model.')
 
+    parser.add_argument('--use_coreml', action='store_true',
```

Comment:
We probably don't want this in the conversion script for now, since this won't work on non-mac computer.
Or can check if the script is running on Mac and throw if not